sdk/core/ts-http-runtime/package.json

```diff
@@ -27,11 +27,28 @@
         "types": "./dist/commonjs/index.d.ts",
         "default": "./dist/commonjs/index.js"
       }
+    },
+    "./internal/logger": {
+      "browser": {
+        "types": "./dist/browser/logger/internal.d.ts",
+        "default": "./dist/browser/logger/internal.js"
+      },
+      "react-native": {
+        "types": "./dist/react-native/logger/internal.d.ts",
+        "default": "./dist/react-native/logger/internal.js"
+      },
+      "import": {
+        "types": "./dist/esm/logger/internal.d.ts",
+        "default": "./dist/esm/logger/internal.js"
+      },
+      "require": {
+        "types": "./dist/commonjs/logger/internal.d.ts",
+        "default": "./dist/commonjs/logger/internal.js"
+      }
     }
   },
   "files": [
     "dist/",
-    "!dist/**/*.d.*ts.map",
```

Comment:
?

Reply:
ah I think I ran dev-tool check and it must have "fixed" this not incorporating the recent updates we made to this section. Will revert (and update dev-tool)